example/tutorial/Algorithms/01_random_numbers/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
-
-KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
-KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+kokkos_include_directories(${CMAKE_CURRENT_BINARY_DIR})
+kokkos_include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
 # This is a tutorial, not a test, so we don't ask CTest to run it.
-KOKKOS_ADD_EXECUTABLE(
-  tutorial_algorithms_01_random_numbers
-  SOURCES random_numbers.cpp
+kokkos_add_executable(
+  tutorial_algorithms_01_random_numbers SOURCES random_numbers.cpp
 )
```

Comment:
I find the previous version more appealing. I guess the formatting wants you to put it on several lines because otherwise the oneliner is too long ?

Reply:
Ideally, something I would prefer something like:

```cmake
kokkos_add_executable(tutorial_algorithms_01_random_numbers
  SOURCES random_numbers.cpp
)
```

Do you think it can be configured this way?